content/zh/docs/tasks/service-catalog/install-service-catalog-using-helm.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 使用 Helm 安装服务目录（Service Catalog）
+content_template: templates/task
+---
+<!--
+---
+title: Install Service Catalog using Helm
+reviewers:
+- chenopis
+content_template: templates/task
+---
+-->
+
+{{% capture overview %}}
+{{< glossary_definition term_id="service-catalog" length="all" prepend="Service Catalog is" >}}
+
+<!--
+Use [Helm](https://helm.sh/) to install Service Catalog on your Kubernetes cluster. Up to date information on this process can be found at the [kubernetes-incubator/service-catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md) repo.
+-->
+使用 [Helm](https://helm.sh/) 在 Kubernetes 集群上安装服务目录。
+要获取有关此过程的最新信息，请浏览 [kubernetes-incubator/service-catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md) 存储库。
```

Comment:
```suggestion
要获取有关此过程的最新信息，请浏览 [kubernetes-incubator/service-catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md) 代码库。
```

Reply:
create a PR for this: https://github.com/k8smeetup/k8s-official-translation/pull/20